lib/removeSourceMap.js

```diff
@@ -0,0 +1,10 @@
+// removes embedded source maps in output css/js files:
+module.exports = (text, type) => {
```

Comment:
why do we need this?  if we don't run through sourcemap process or enable debugging, this shouldn't get added, correct?